### PR TITLE
Apr 2655 appeal outcome view suresh

### PR DESCRIPTION
**Pull request by sureshkuruva1**

### CODE REVIEW

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    await Html.RenderPartialAsync("_AppealSuccessComments.cshtml", Model);
+}
+else if(appealStatus == AppealStatus.InProgress)
+{
+    await Html.RenderPartialAsync("_AppealInprogressComments.cshtml", Model);
+}
+else
+{
+    await Html.RenderPartialAsync("_AppealOutcomeComments.cshtml", Model);
+}
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml:85]: Html.RenderPartialAsync is not MVC core compliant, please use the appropriate tag helper

**Review by martin-lush — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml (hunk 1 above)
Comment (sureshkuruva1) [reply src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml]: done

**Review by martin-lush — APPROVED** [review]

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml (hunk 1 above)
Comment (sureshkuruva1) [reply src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml]: added partial tag <parrtial> instead Html.PartialAsync , as partialAsync will not render for conditional based things

Comment (sureshkuruva1) [reply src/SFA.DAS.RoatpOversight.Web/Views/Shared/_AppealOutcomeDetails.cshtml]: @martin-lush

**Pull request merged**